### ISSUE TITLE
Private room on dedicated server

### ISSUE THREAD

**Issue by Triplkrypl** [Kromster80/kam_remake#384]
I created for my friends dedicated server, becouse if we want play alone and it is annoing when other players conecting into our room.
Now i know then we can set pasword on any empty room on any server and this solve the problem.
It is good solution, but we have to on each connect to empty room lock it, if i understood your code on github correctly.

Can you add some funkcionaliti on server config where can be set password for one or more rooms and this password will be active for server run-time.

This will be more comfortable, i can with my friends share one password witch is not change and other players who will play on my server can connect to others rooms witch is auto created as now.

**Comment by Kromster80** [comment]
So what you suggest is to allow to set default passwords on dedicated server rooms (from INI file)?

**Comment by Triplkrypl** [comment]
Yes if room is empty will required password (loaded from INI) for connect and this password can not be changed by host player.
Best solution will be that password in INI file will bound on room id, other rooms can be public.
